Documentation/Index.rst

```diff
@@ -52,4 +52,3 @@ Core Documentation Code Examples
    Introduction/Index
    Users/Index
    Maintainers/Index
-
```

Comment:
Imho files better end with an LF.

Reply:
I always thought empty lines on the end of a file are evil...

Reply:
It seems like LFs at the end of files get automatically added, if you look here: https://github.com/linawolf/TYPO3CMS-Code-Examples/edit/feature-docu/Documentation/Index.rst the LF is still there. Same is locally in my browser. For the same reason it seems to be impossible to make a second commit that would remove this change.

Reply:
Your link does not work, it's an edit link and I ofc have no write access to your fork.

Reply:
can you see this one? it also has the empty line in line 55: https://raw.githubusercontent.com/TYPO3-Documentation/TYPO3CMS-Code-Examples/22521c4e0049eaf22af8d6266e5a9ffe3ab6e658/Documentation/Index.rst